__tests__/fixtures/install/install-prod-deduped-direct-dev-dep/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "a": "file:a"
+    "a": "file:./a"
   },
   "devDependencies": {
-    "b": "file:b"
+    "b": "file:./b"
```

Comment:
Without this fix the test was failing.
That is strange and needs a bit more investigation later on